src/main/java/com/rsomeara/factorio/modlauncher/Controller.java

```diff
@@ -1,12 +1,45 @@
-package com.rsomeara.factorio.modlauncher;
-
-import javafx.fxml.FXML;
-import javafx.scene.control.Button;
-
-public class Controller {
-
-	@FXML
-	private Button goButton;
-
-}
-
+package com.rsomeara.factorio.modlauncher;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+import javafx.scene.control.TreeView;
+
+public class Controller {
+
+	@FXML
+	private Button launchButton;
+	
+	@FXML
+	private Button newButton;
+	
+	@FXML
+	private TreeView modPackTree;
+	
+	@FXML
+	private TextField nameField;
+	
+	@FXML
+	private ListView modsList;
+	
+	//on event methods
+	

```

Comment:
I'd put @FXML on these methods to make it clear what calls them, and move those comments into javadoc comments :)